### PR TITLE
Chore/mobile hook reoganization

### PR DESCRIPTION
**Pull request by dsilva609**
## ✏️ Description
- moves all hooks for mobile to a new hooks folder

## 🔬 To Test
1. Run through mobile app and it should behave as normal

## 📝 Checklist

- [ ] I have performed a self-review of my code
- [ ] I have tested my code works in the project (nothing breaks)
- [ ] I have commented my code in hard-to-understand areas
- [ ] New and existing tests and linting pass locally with my changes
- [ ] I have made corresponding changes to the documentation (new envs?)
- [ ] For PR's with design updates/changes, I have consulted with the product designer and made all necessary adjustments in Figma

### CODE REVIEW

**Review by calebpanza — CHANGES_REQUESTED** [review]
Can you please update explicit imports to be relative `import ... from 'hooks'` in the following directories:

`components`
`features`
`screens`

**Comment by dsilva609** [comment]
@calebpanza updated imports to be relative for hooks

File: templates/mobile/src/hooks/useFlagReaction.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { useActivityContext } from './StreamActivity';
-import { useStreamFeedReaction } from 'utils';
+import { useActivityContext } from '../components/StreamActivity/StreamActivity';
```
Comment (calebpanza) [inline templates/mobile/src/hooks/useFlagReaction.js:1]: `from 'components/StreamActivity/StreamActivity'` for consistent relative imports

File: templates/mobile/src/hooks/usePrayReaction.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import uniqBy from 'lodash/uniqBy';
 
-import { useCurrentPersonProfile } from 'utils';
+import { useCurrentPersonProfile, useStreamFeedReaction } from './index';
 
-import { useActivityContext } from './StreamActivity';
-import { useStreamFeedReaction } from 'utils';
+import { useActivityContext } from '../components/StreamActivity/StreamActivity';
```
Comment (calebpanza) [inline templates/mobile/src/hooks/usePrayReaction.js:5]: Relative import `components/StreamActivity/StreamActivity`

File: templates/mobile/src/hooks/useReplyReaction.js [hunk 3]
```diff
@@ -1,7 +1,5 @@
-import { useCurrentPersonProfile } from 'utils';
-
-import { useActivityContext } from './StreamActivity';
-import useStreamFeedReaction from './useStreamFeedReaction';
+import { useActivityContext } from '../components/StreamActivity/StreamActivity';
```
Comment (calebpanza) [inline templates/mobile/src/hooks/useReplyReaction.js:1]: Relative import `components/StreamActivity/StreamActivity`

**Review by calebpanza — CHANGES_REQUESTED** [review]

File: templates/mobile/src/hooks/usePrayReaction.js (hunk 2 above)
Comment (dsilva609) [reply templates/mobile/src/hooks/usePrayReaction.js]: updated

File: templates/mobile/src/hooks/useReplyReaction.js (hunk 3 above)
Comment (dsilva609) [reply templates/mobile/src/hooks/useReplyReaction.js]: updated

**Review by calebpanza — APPROVED** [review]

**Pull request merged**